src/datastructures.apib

```diff
@@ -66,3 +66,21 @@
         + end_of_month - End of the month of Xth day after the invoice date
         + after_invoice_date - X days after invoice date
 + days (number, optional) - Modifier "X" for the above statements. Not required when type is 'cash'.
+
+## CustomField (object)
++ definition (object)
+    + type: `customFieldDefinition` (string)
+    + id: `bf6765de-56eb-40ec-ad14-9096c5dc5fe1` (string)
++ One Of
```

Comment:
Indentation issue?

Reply:
Why would there be an indentation issue? 🤔

Reply:
Sorry, my bad. Looked weird to see properties without indentation.

Reply:
confused with the double `+` signs? 😄 It confused me too for a second

 :ok: by @WouterSioen on 30/03/2018 12:07:24